### PR TITLE
Update the Format of IBM Credential V2

### PR DESCRIPTION
**Pull request by tbcdebug**
Replaced the old version of accessing IBMQ provider

### CODE REVIEW

File: ch-algorithms/quantum-fourier-transform.ipynb [hunk 1]
```diff
@@ -633,7 +633,8 @@
    ],
    "source": [
     "# Use the tenerife (ibmqx4) device\n",
-    "backend = IBMQ.get_backend('ibmqx4')\n",
+    "IBMQ.get_provider(hub='ibm-q')\n",
+    "backend = provider.get_backend('ibmqx4')\n",
```
Comment (aasfaw) [inline ch-algorithms/quantum-fourier-transform.ipynb]: Can you please replace this one with `ibmq_vigo`? `ibmqx4` is no longer available.

File: ch-algorithms/quantum-phase-estimation.ipynb [hunk 2]
```diff
@@ -572,7 +573,7 @@
     }
    ],
    "source": [
-    "backend = IBMQ.get_backend('ibmq_20_tokyo')\n",
+    "backend = provider.get_backend('ibmq_20_tokyo')\n",
```
Comment (aasfaw) [inline ch-algorithms/quantum-phase-estimation.ipynb]: Can you please replace this one with `ibmq_vigo`? `ibmq_20_tokyo` is a premium device and may not be available to all readers.

File: ch-algorithms/quantum-fourier-transform.ipynb [hunk 3]
```diff
@@ -633,7 +633,8 @@
    ],
    "source": [
     "# Use the tenerife (ibmqx4) device\n",
```
Comment (aasfaw) [inline ch-algorithms/quantum-fourier-transform.ipynb]: And remove this comment line to `Use the IBMQ Vigo device with 5 qubits`

**Comment by aasfaw** [comment]
This is ready to go after those minor changes. Thanks @tbcdebug.

Fixes #14

**Comment by tbcdebug** [comment]
Alright, I have made the changes. Thanks for the updates on these devices! @aasfaw

**Pull request merged**